### DISCUSSION TITLE
Already encrypted fields

### DISCUSSION THREAD

**Discussion by iambateman** [spatie/laravel-ciphersweet#4] [Q&A]
This looks amazing, thanks for your work on it!

Quick question...the readme recommends using a cyphersweet command to generate the encryption key. 

Does this work with standard laravel encryption casting?

I have a database that already encrypts data, and I'd like to add this package if possible!

**Comment by freekmurze** [comment]
Thanks for the kind words on our work.

> Does this work with standard laravel encryption casting?

No, we use another way of encrypting and handling encrypted values.
